src/components/flat-table/flat-table-body-draggable/flat-table-body-draggable.test.tsx

```diff
@@ -22,51 +127,34 @@ describe("FlatTableBodyDraggable", () => {
     loggerSpy.mockRestore();
   });
 
-  describe("when a data prop is added", () => {
-    test("should be added to the FlatTableBody", () => {
-      render(
-        <FlatTable>
-          <FlatTableBodyDraggable data-role="test">
-            <FlatTableRow key={0} id={0}>
-              <FlatTableCell>UK</FlatTableCell>
-            </FlatTableRow>
-            <FlatTableRow key={1} id={1}>
-              <FlatTableCell>Germany</FlatTableCell>
-            </FlatTableRow>
-          </FlatTableBodyDraggable>
-        </FlatTable>
-      );
-
-      expect(screen.getByTestId("test")).toBeTruthy();
-    });
+  it("sets data-role on table body when passed as a prop", () => {
+    render(
+      <FlatTable>
+        <FlatTableBodyDraggable data-role="test">
+          <FlatTableRow key={0} id={0}>
+            <FlatTableCell>UK</FlatTableCell>
+          </FlatTableRow>
+          <FlatTableRow key={1} id={1}>
+            <FlatTableCell>Germany</FlatTableCell>
+          </FlatTableRow>
+        </FlatTableBodyDraggable>
+      </FlatTable>
+    );
+
+    expect(screen.getByRole("rowgroup")).toHaveAttribute("data-role", "test");
   });
 
   describe("drag and drop functionality", () => {
-    beforeEach(() => {
-      render(
-        <FlatTable>
-          <FlatTableBodyDraggable>
-            <FlatTableRow key="0" id={0}>
-              <FlatTableCell>Row one</FlatTableCell>
-            </FlatTableRow>
-            <FlatTableRow key="1" id={1}>
-              <FlatTableCell>Row two</FlatTableCell>
-            </FlatTableRow>
-            <FlatTableRow key="2" id={2}>
-              <FlatTableCell>Row three</FlatTableCell>
-            </FlatTableRow>
-          </FlatTableBodyDraggable>
-        </FlatTable>
-      );
-    });
-
-    test("on initial render the rows are in the correct order", () => {
+    it("on initial render the rows are in the correct order", () => {
```

Comment:
suggestion: this doesn't really read quite right to me, maybe something like the below would work

```
it("should have roes in correct order on initial render")